crates/sui-json-rpc/src/unit_tests/rpc_server_tests.rs

```diff
@@ -110,7 +110,7 @@ async fn test_public_transfer_object() -> Result<(), anyhow::Error> {
     let gas = objects.clone().last().unwrap().object().unwrap().object_id;
 
     let transaction_bytes: TransactionBlockBytes = http_client
-        .transfer_object(address, obj, Some(gas), 10_000.into(), address)
```

Comment:
I find it hard to believe that these tests worked with gas budget set to 10000. I would imagine the transaction would fail with out of gas. Did the transactions actually succeed? Or were we using dummy RGP of 1 MIST or something?